apps/financial-aid/api/src/app/modules/application/application.resolver.ts

```diff
@@ -18,19 +15,16 @@ import {
   UpdateApplicationInput,
   CreateApplicationEventInput,
   UpdateApplicationInputTable,
+  ApplicationInput,
+  AllApplicationInput,
 } from './dto'
-import { JwtGraphQlAuthGuard } from '@island.is/financial-aid/auth'
-
-import { ApplicationInput, AllApplicationInput } from './dto'
-
 import {
   Application,
   ApplicationFilters,
   UpdateApplicationTableResponseType,
   UpdateApplicationResponseType,
 } from '@island.is/financial-aid/shared/lib'
 
-@UseGuards(JwtGraphQlAuthGuard)
 @Resolver(() => ApplicationModel)
```

Comment:
Should this resolver not be guarded?

Reply:
True, I'll add `IdsUserGuard` for this resolvers and others 😃